### PR TITLE
DOCS-573 Remove unsupported Viridian features

### PR DESCRIPTION
**Pull request by angelasimms**
Also includes removal of pricing information.

The following changes **_are not_** included:

- Changes to Viridian Dedicated product description, banners etc. ([DOCS-560](https://hazelcast.atlassian.net/browse/DOCS-560))
- WAN replication (dependency on UI change - already approved: [DOCS-554](https://hazelcast.atlassian.net/browse/DOCS-554)).
- VPC peering for AWS (in progress as part of a separate ticket: [DOCS-580](https://hazelcast.atlassian.net/browse/DOCS-580). VPC peering section has been removed until this content is ready.
- Creation of new 'Clients` section, i.e. moving sections into the correct order to reflect the workflow [DOCS-553](https://hazelcast.atlassian.net/browse/DOCS-553)

[DOCS-554]: https://hazelcast.atlassian.net/browse/DOCS-554?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOCS-580]: https://hazelcast.atlassian.net/browse/DOCS-580?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DOCS-560]: https://hazelcast.atlassian.net/browse/DOCS-560?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DOCS-553]: https://hazelcast.atlassian.net/browse/DOCS-553?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lovanchetty — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/authorize-connections.adoc [hunk 1]
```diff
@@ -9,10 +9,7 @@ If you're using a {hazelcast-cloud} Serverless cluster, you can add your client
...
 == {hazelcast-cloud} Dedicated
 
-If you're using a {hazelcast-cloud} Dedicated cluster, you must authorize your clients to connect to your cluster. {hazelcast-cloud} Dedicated clusters are private by default to prevent attacks such as denial of service. To authorize clients, you can do the following:
+If you're using a {hazelcast-cloud} Dedicated cluster, you must authorize your clients to connect to your cluster. {hazelcast-cloud} Dedicated clusters are private by default to prevent attacks such as denial of service. To authorize clients, you can do the following add client IP addresses to a xref:ip-white-list.adoc[whitelist].
 
-- Add client IP addresses to a xref:ip-white-list.adoc[whitelist].
-- Set up xref:vpc-peering.adoc[PrivateLink or VPC peering].
-
-TIP: You should use PrivateLink or VPC peering if your clients' IP addresses are not static or if you want to limit your cluster's exposure to the public Internet.
+TIP: You must use VPC peering if your clients' IP addresses are not static or if you want to limit your cluster's exposure to the public Internet.
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/authorize-connections.adoc:14]: Checked with Atil and the endpoints I private. I will remove the misleading text.

File: docs/modules/ROOT/pages/serverless-cluster.adoc [hunk 2]
```diff
@@ -74,7 +74,7 @@ For example, if you configure a map with five backups, but your cluster has only
 
 {hazelcast-cloud} Serverless clusters have the following heap memory limits:
 
-- Production clusters: 1 GB memory per cluster member
+- Production clusters: 0.5 GiB memory per cluster member
 - Development clusters: 300 MB per cluster
 
 Exceeding these limits can lead to cluster failure.
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/serverless-cluster.adoc:80]: @lovanchetty and @FinnLiuHZ. Attaching ticket which has all the relevant details: https://hazelcast.atlassian.net/browse/HZC-3956

File: docs/modules/ROOT/pages/overview.adoc [hunk 3]
```diff
@@ -21,17 +21,15 @@ Hazelcast {hazelcast-cloud} applications run in a client/server topology:
...
 |===
-|Edition|Description|Pricing
+|Edition|Description
 
 |xref:serverless-cluster.adoc[{hazelcast-cloud} Serverless]
 |{hazelcast-cloud} Serverless clusters auto-scale, depending on the amount of data you store.
-|Free to store up to 2 GiB in any {hazelcast-cloud} Serverless cluster in your account, then $0.10 USD per GiB.
 
 |xref:dedicated-cluster.adoc[{hazelcast-cloud} Dedicated] (coming soon)
 |{hazelcast-cloud} Dedicated clusters give you fine-grained control over the hardware and resources available to your cluster. It's your responsibility to scale {hazelcast-cloud} Dedicated clusters when you need more resources.
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/overview.adoc:32]: Updated.

File: docs/modules/ROOT/pages/overview.adoc [hunk 4]
```diff
@@ -21,17 +21,15 @@ Hazelcast {hazelcast-cloud} applications run in a client/server topology:
...
+[cols="1a,2a"]
 |===
-|Edition|Description|Pricing
+|Edition|Description
 
 |xref:serverless-cluster.adoc[{hazelcast-cloud} Serverless]
 |{hazelcast-cloud} Serverless clusters auto-scale, depending on the amount of data you store.
-|Free to store up to 2 GiB in any {hazelcast-cloud} Serverless cluster in your account, then $0.10 USD per GiB.
 
 |xref:dedicated-cluster.adoc[{hazelcast-cloud} Dedicated] (coming soon)
```
[the remark this answers is not in the document]
Comment (angelasimms) [reply docs/modules/ROOT/pages/overview.adoc:31]: Updated.

**Review by lovanchetty — APPROVED** [review]

**Pull request merged**